packages/paste-website/src/pages/components/table/api.mdx

```diff
@@ -0,0 +1,80 @@
+export const meta = {
+  title: 'Table - API',
+  package: '@twilio-paste/table',
+  description: 'Tables are used to display information that is meant to be compared across columns and rows.',
```

Comment:
Same here - did you intentionally change this?

> A Table is a collection of information displayed across columns and rows.

Reply:
If so can you update the one in the package.json to match

Reply:
gotcha. would be a great nice-to-have to only write the package description in 1 place

Reply:
Yeah, I think there are generally improvements to be made where we can auto generate more of these component or package packages, or at least parts of them. They're not very templatized and updating them is a super big pain